docs/UserGuide.md

```diff
@@ -205,6 +227,20 @@ Examples:
 * `add t/doctor i/D123 c/11112222`
 * `add t/doctor i/D101 s/orthopaedics`
 
+### Viewing the list of doctors `list -d -c -s`
```

Comment:
Maybe we should do `list t/doctor` as there is `list t/activity` to make the format uniform and the flag I think can be ignored for now because this means changing the UI extensively for every different combi of flags (for eg list -d -c -s and -d -s -c, should we display contact then specialty or specialty then contact)

Reply:
Yup I agree. I have made the changes.